x-pack/plugins/fleet/public/components/agent_enrollment_flyout/hooks.tsx

```diff
@@ -210,12 +210,15 @@ export function useGetCreateApiKey() {
   const core = useStartServices();
 
   const [apiKey, setApiKey] = useState<string | undefined>(undefined);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
I added a loading state, as it was weird to me to not have feedback while the API key is creating